crates/api_common/src/utils.rs

```diff
@@ -551,17 +551,13 @@ pub async fn purge_post_images(
 
 /// Delete a local_user's images
 async fn delete_local_user_images(person_id: PersonId, context: &LemmyContext) -> LemmyResult<()> {
```

Comment:
```suggestion
/// Delete local images attributed to a person
async fn delete_local_user_images(person_id: PersonId, context: &LemmyContext) -> LemmyResult<()> {
```
at least the comment should be adjusted, if not the function name as well.